packages/react-reconciler/src/__tests__/ReactExpiration-test.js

```diff
@@ -320,13 +313,10 @@ describe('ReactExpiration', () => {
     }
 
     // Initial mount
-    if (gate(flags => flags.enableSyncDefaultUpdates)) {
```

Comment:
Most of these gate checks just switch to use transitions if we're sync by default. But we can just always use transitions, since that what we'll need to do long term to do these tests anyway.